src/main/java/org/cactoos/text/Concatenated.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Text;
+import org.cactoos.iterable.IterableOf;
+import org.cactoos.scalar.LengthOf;
+
+/**
+ * Concatenate a texts.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @since 0.46
+ */
+public class Concatenated extends TextEnvelope {
+
+    /**
+     * Ctor.
+     * @param txts Texts to be concatenated
+     */
+    public Concatenated(final Text... txts) {
+        this(new IterableOf<Text>(txts));
```

Comment:
@kokodyn Shouldn't it be simply `new Joined(new TextOf(""), txts)`?

Reply:
@andreoss I was thinking about performence a bit.  It is not so easy to decide what is faster StringJoiner(which is used in background in Joined) or StringBuilder so I think we can use Joined at now, because it is easier and more elegant.